src/27-ipfs-at-the-vatican/index.md

```diff
@@ -0,0 +1,158 @@
+
+In the meantime, to confirm that you're actually retrieving content that any IPFS node could resolve, you can replace the ipfs.io portion of any url with the hostname of any IPFS gateway. There is a [thread in the IPFS discussion forum](https://discuss.ipfs.io/t/curated-list-of-ipfs-gateways/620) where people are gathering a list of gateways you could test out.
+
+## The Second Demo: Serverless Collaborative Annotations on IPFS
+
+Our second demo is more involved and it requires more explanation. The demo focuses on allowing people to annotate images and sync those annotations across the peer-to-peer network without relying on an annotation server. To support this real-time collaborative sharing of data we use [CRDTs](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type) and [IPFS pubsub](../25-pubsub/).
+
+One obstacle to enabling annotations of IIIF content is the fact that nobody wants to run an annotation server. Institutions want to let their own patrons and curators create annotations. Though they are also interested in seeing the annotations other people create, they don't want to get stuck storing the whole world's annotations indefinitely. That would be like your local public library committing to store all the notes anyone ever made about the books in their collection rather than letting everyone take their own notes home with them and store those notes as they see fit.
+
+Peer to peer tools are the right tech to apply here. With the demo we set up, each node on the network holds its own copy of the annotations. The user of that node decides which streams of annotations they will publish to and subscribe from. Groups of people can share their annotations as they see fit. Third parties like the library can follow or ignore public streams, choosing which content to keep a copy of.
+
+This is an example of the permissionless web — where nobody needs permission to create content or share it. There is no annotation server, so there is no gatekeeper deciding who gets to create annotations; everyone is free to create content as they see fit.
+
+This peer-to-peer approach also gives us an example of the role for cultural institutions on the decentralized web. If each person or group is holding their own copies of the data they deem important, you inevitably need someone to coordinate the effort to ensure ongoing access to those data, to ensure that people will find/discover the content, and to ensure that the content is preserved for the long term. This is exactly the role that Libraries, Archives and Museums exist to fill.
```

Comment:
> If each person or group is holding their own copies of the data they deem important, you inevitably need someone to coordinate the effort to ensure ongoing access to those data, to ensure that people will find/discover the content, and to ensure that the content is preserved for the long term.

How about: If each person or group is holding their own copies of the data they deem important, you inevitably need someone to coordinate the effort. You need to ensure the data can be accessed, that people can find/discover it, and that it's preserved for the long term.